### ISSUE TITLE
.length not always working on strings

### ISSUE THREAD

**Issue by 2pha** [endel/js2php#30]
eg.
Input:
```
var st = 'testing';
var hmm = st.length;
console.log(st.length);
```
Output:
```
$st = 'testing';
$hmm = strlen($st);
$console->log(count($console));
```

I am working on a fix.

**Comment by 2pha** [comment]
This is partially fixed in my other pull request
https://github.com/endel/js2php/pull/29

**Comment by endel** [comment]
Thanks for the pull-request, @2pha! 👏

Just released `0.1.1` with your fixes.
